lib/hyperlogs/hyperlogs/rollbar_utilities.py

```diff
@@ -0,0 +1,98 @@
+"""Utilities for rollbar."""
+import datetime
+import getpass
+import logging
+import os
+import platform
+import sys
+
+import rollbar
+from dotenv import load_dotenv
+
+load_dotenv("rollbar.env")
+ROLLBAR_API_KEY = os.getenv("ROLLBAR_API_KEY")
+
+env_details = {
```

Comment:
I think these should live in the crash report, not in the rollbar only. That way we consolidate features & information to one place.

Reply:
good idea.